formats/json/commonMain/src/kotlinx/serialization/json/internal/FormatLanguage.kt

```diff
@@ -0,0 +1,45 @@
+package kotlinx.serialization.json.internal;
+
+import kotlinx.serialization.InternalSerializationApi
+
+/**
+ * Multiplatform analogue of `org.intellij.lang.annotations.Language` annotation.
+ *
+ * An alias is used instead of classes, because the actual class in the JVM will overload the classes from the stdlib -
```

Comment:
Do you mean 'classes' or a single 'class'?

Reply:
single class in this case